py3-2to3.yaml

```diff
@@ -0,0 +1,32 @@
+# Generated from https://pypi.org/project/2to3/
+package:
+  name: py3-2to3
+  version: "1.0"
+  epoch: 0
+  description: Adds the 2to3 command directly to entry_points.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 958bc212c928bbdcbc778b72528e0a39ae4ee8040eda6af6c3b5dd640c98ce6d
+      uri: https://files.pythonhosted.org/packages/70/06/9bd891a16b16572e42d124627048a673f9e201c490ef1638b689a94cacd7/2to3-${{package.version}}.tar.gz
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
```

Comment:
No update block?

Reply:
Sounds like 2to3 is installed along with python itself, and this project just makes it available as a script?

2to3 is also deprecated and will be removed in 3.13, so I don't think this will get updates.